.github/coding_guidelines/LABELING_GUIDELINES.md

```diff
@@ -0,0 +1,27 @@
+# PR Labeling Guidelines
+To maintain a consistent and efficient development workflow, we have set specific label guidelines for all pull requests (PRs). Please ensure you adhere to the following instructions:
+
+### Mandatory team labels:
+- **Internal Developers**: Every PR raised by an internal developer must include a label prefixed with `team-` (e.g., `team-mobile-ux`, `team-mobile-platform`, etc.). This indicates the respective internal team responsible for the PR.
+
+- **External Contributors**: PRs from contributors outside the organization must have the `external-contributor` label.
+
+It's essential to ensure that PRs have the appropriate labels before they are considered for merging.
+
+### Optional QA labels:
+- **needs-qa**: If the PR includes a new features, complex testing steps, or large refactors, this label must be added to indicated PR requires a full manual QA prior being merged and added to a release.
+
+### Labels prohibited when PR needs to be merged:
+Any PR that includes one of the following labels can not be merged:
```

Comment:
```suggestion
Any PR that includes one of the following labels cannot be merged:
```

Reply:
Both "cannot" and "can not" are technically correct.
The latter is used to emphasize the "not" part which, was the intention here.